ui/apps/platform/src/Containers/MainPage/Navigation/NavigationSidebar.tsx

```diff
@@ -417,7 +341,8 @@ function NavigationSidebar({
                             const hasChildMatchPath = children.some(
                                 (childDescription) =>
                                     childDescription.type === 'link' &&
-                                    Boolean(matchPath(pathname, childDescription.path))
+                                    (Boolean(matchPath(pathname, childDescription.path)) ||
+                                        isActiveLink(pathname, childDescription))
```

Comment:
Adds the ability to determine if a parent link is active by its description's declarative `isActive` function.

This will be needed when the subnav is implemented, because the children of the parent may not include all links that are accessible from the parent.